projects/commudle-admin/src/app/feature-modules/user-follow/user-follow.component.scss

```diff
@@ -0,0 +1,33 @@
+@import '/projects/commudle-admin/src/assets/styles/variables';
+
+.user-follow-container {
+  .follow-btn {
+    border: none;
+    border-radius: $border-10;
+    padding: $space-8;
+    height: 32px;
+    font-size: $fs-h6;
+    font-weight: $fw-400;
+    text-transform: none;
+    color: $neon-blue;
+    background-color: rgba($neon-blue, 0.2);
+
+    &:hover {
+      background-color: rgba($neon-blue, 0.2);
+    }
+
+    nb-icon {
+      margin-right: $space-4;
+    }
+
+    &.active {
+      border: 1px solid rgba($neon-blue, 0.2);
+      background-color: $white;
+    }
+  }
+}
```

Comment:
If this button is similar to some of the other buttons, can we have this css commonly written and create a common class?

Reply:
Right now we are using this styling in only two places and the styling of hovering over follow button is being used only here, so is it advisable to create a common class for this?